trio/socket.py

```diff
@@ -56,25 +56,25 @@
 # expose these functions to trio.socket
 from socket import (
     gaierror as gaierror,
-    herror as herror,
     gethostname as gethostname,
-    ntohs as ntohs,
+    herror as herror,
     htonl as htonl,
     htons as htons,
     inet_aton as inet_aton,
     inet_ntoa as inet_ntoa,
-    inet_pton as inet_pton,
     inet_ntop as inet_ntop,
+    inet_pton as inet_pton,
+    ntohs as ntohs,
 )
 
 # not always available so expose only if
 if sys.platform != "win32" or not _t.TYPE_CHECKING:
     try:
         from socket import (
-            sethostname as sethostname,
+            if_indextoname as if_indextoname,
             if_nameindex as if_nameindex,
             if_nametoindex as if_nametoindex,
-            if_indextoname as if_indextoname,
+            sethostname as sethostname,
         )
     except ImportError:
         pass
```

Comment:
The windows CI fails is because `pyright` now doesn't see `if_indextoname`, `if_nameindex` or `if_nametoindex` on windows ... but it sees `sethostname`? And why did it "work" in the first place, since `sys.platform == "win32"` and `_t.TYPE_CHECKING == TRUE` it shouldn't have ever seen these symbols if it's parsing the `if` statement.

Regardless, I don't think it's an issue, and I don't care to dig deeper into it atm, so I'll just `Remove` those in the test.